Dockerfile_local

```diff
@@ -8,6 +8,8 @@ RUN --mount=type=secret,id=aws,target=/root/.aws/credentials aws s3 cp s3://com.
 # Maven target to get all dependencies and build
 FROM maven:3.6.2-jdk-11 as packager
 # build arguments
+ARG PROJECT_FOLDER=Openfire
```

Comment:
It's a good idea to also add this to the README - you have such a nice guide how to build an image. It would be a shame to miss one important argument.

Reply:
true